gooddata-sdk/gooddata_sdk/compute/model/execution.py

```diff
@@ -335,6 +347,7 @@ def read_result(self, limit: Union[int, list[int]], offset: Union[None, int, lis
             limit=_limit,
             _check_return_type=False,
             _return_http_data_only=False,
+            x_gdc_cancel_token=self.cancel_token,
```

Comment:
Be careful here. x_gdc_cancel_token should be string, but there is a difference between Python string boolean ("True", "False") and other string boolean representation ("true", "false"). Make sure that it will work.